### PR TITLE
Update wordcount example to use the 0.9.5 mapreduce dep

### PR DESCRIPTION
**Pull request by igorbernstein2**

### CODE REVIEW

File: java/dataproc-wordcount/pom.xml [hunk 1]
```diff
@@ -96,30 +92,15 @@
...
-      <version>1.1.33.Fork19</version>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-client</artifactId>
+      <version>2.7.3</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>com.google.cloud.bigtable</groupId>
       <artifactId>bigtable-hbase-mapreduce</artifactId>
       <version>${bigtable.version}</version>
```
Comment (sduskis) [inline java/dataproc-wordcount/pom.xml]: Should we add a <classifier>shaded</classifer> tag here?

File: java/dataproc-wordcount/pom.xml [hunk 2]
```diff
@@ -32,15 +32,11 @@
...
     <bigtable.projectID>YOUR_PROJECT_ID_HERE</bigtable.projectID>
     <bigtable.instanceID>YOUR_INSTANCE_ID_HERE</bigtable.instanceID>
 
-    <bigtable.version>0.9.4</bigtable.version>
-    <hbase.version>1.2.1</hbase.version><!-- may need to match Dataproc HBase version -->
-    <hbase.if>1.2</hbase.if> <!-- hbase interface - it changed between 1.0 and 1.1.x -->
-
-    <hadoop.version>2.4.1</hadoop.version>
-    <compat.module>hbase-hadoop2-compat</compat.module>
+    <bigtable.version>0.9.5-SNAPSHOT</bigtable.version>
```
Comment (sduskis) [inline java/dataproc-wordcount/pom.xml]: There's a duplicate <bigtable.version> tag.  I think we should wait until 0.9.5 to commit this change.

File: java/dataproc-wordcount/pom.xml (hunk 1 above)
Comment (igorbernstein2) [inline java/dataproc-wordcount/pom.xml]: I don't think it adds or subtracts anything. I left it off to keep things simpler

File: java/dataproc-wordcount/pom.xml (hunk 2 above)
Comment (igorbernstein2) [inline java/dataproc-wordcount/pom.xml]: Oops, forgot to remove test code :disappointed:

**Comment by lesv** [comment]
Ping ??

**Comment by sduskis** [comment]
this is obsolete

**Pull request closed without merging**